config/jobs/kubernetes/test-infra/test-infra-periodics.yaml

```diff
@@ -24,6 +24,37 @@ periodics:
     testgrid-tab-name: ci-bazel
     description: Runs bazel test //... on the test-infra repo every hour
 
+# This job is used as a heartbeat health check of the Prow instance's ability to run jobs.
+# Alerts expect it to run every 10 mins and will fire after 20 mins without a successful run.
+# Please keep this in sync with the `pull-test-infra-prow-checkconfig` job
```

Comment:
Can you specify (in the code comment) what "in sync" means?

Reply:
Discussed out of band. I think it will be apparent to anyone making changes whether the change should be reflected based on the comments.